lib/test.gi

```diff
@@ -20,19 +20,18 @@ InstallGlobalFunction(ParseTestInput, function(str, ignorecomments)
   ign := [];
   i := 1;
   while i <= Length(lines) do
-    if i = 1 and  Length(lines[1]) > 0 and lines[1][1] = '#' then
+    if i = 1 and (lines[1] = "" or lines[1][1] = '#') then
```

Comment:
Note that this line creates a temporary empty string object each time it is executed, while the old version with `Length` does not create any temporaries.

Reply:
Usually I'd assume function calls cost, but I agree that Length on strings is cheap. I don't really care about one string, but I have made the change so I can get this merged.